### DISCUSSION TITLE
Is it possible to add font-feature-settings to theme.json

### DISCUSSION THREAD

**Discussion by CGTS** [WordPress/gutenberg#47127] [Q&A] [answered]
I have a variable font I've added to theme.json and I would like the following to be included in the css:

`font-feature-settings: "cv02","cv03","cv04","cv11";`

Is there a way to do this through the theme.json file?

**Comment by t-hamano** [comment, marked as answer]
Hi.

When specifying a font family, its properties can also be defined. Like this:

```json
{
	"$schema": "https://schemas.wp.org/trunk/theme.json",
	"version": 2,
	"settings": {
		"typography": {
			"fontFamilies": [
				{
					"fontFace": [
						{
							"fontFamily": "Source Serif Pro",
							"fontFeatureSettings": "some-value",
							"src": [
								"file:./assets/fonts/source-serif-pro/SourceSerif4Variable-Roman.ttf.woff2"
							]
						}
					],
					"fontFamily": "\"Source Serif Pro\", serif",
					"name": "Source Serif Pro",
					"slug": "source-serif-pro"
				}
			]
		}
	}
}

```

From this definition, the following styles are output:

```html
<style id="wp-fonts-local">
@font-face{
	font-family:"Source Serif Pro";
	font-style:normal;font-weight:400;
	font-display:fallback;src:url('/wp-content/themes/emptytheme/assets/fonts/source-serif-pro/SourceSerif4Variable-Roman.ttf.woff2') format('woff2');
	font-feature-settings:some-value;
}
</style>
```

Is this the solution you are looking for?

**Reply by CGTS** [reply]
You're right. I wasn't seeing it in the style inspector, but it does show up in the @font-face rules. Seems like I misunderstood how it was being generated. Thanks for your comment it helped clear up what was happening.
